SwiftAppTemplate/Dummy-Use&Delete/SettingsScreen.swift

```diff
@@ -20,8 +23,30 @@ struct SettingsScreen: View {
             VStack(spacing: 30){
                 
                 Header(text: "Settings", hasBackButton: true, onBackArrowClick: {dismiss()})
+         
+                TitleValueView(title: "Name", value: viewModel.userName)
+                
+                TitleValueView(title: "Email", value: viewModel.userEmail)
+                
+                TitleValueView(title: "Gender", value: getLocalString(viewModel.gender))
+                
+                TitleValueView(title: "DateOfBirth", value: viewModel.dob)
+        
+                TitleValueView(title: "Country", value: viewModel.country)
+
+                TitleValueView(title: "Language", value: viewModel.language)
+
                 
-                customTextFields
+                TextButton(onClick: {
```

Comment:
This text button can be put in a variable.

Reply:
fixed